src/deferred.js

```diff
@@ -385,5 +394,12 @@ jQuery.extend( {
 	}
 } );
 
+function deferredTick() {
+	processQueueTimeout = undefined;
+	jQuery.each( processQueue.splice( 0 ), function( _, process ) {
```

Comment:
> Maybe we've reached a choice between the lesser of two evils

Are the possibly-redundant timeouts from our current ungrouped implementation really so evil? I'd sure prefer them to masking errors, and we don't have the option of rejecting because only standard callback completion is permitted to do that.